README.md

```diff
@@ -43,32 +43,24 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
 
 Wait for about a minute for MarkLogic to initialize before checking the ports.
 
-### Persistent Data Directory
+### Persistent Data Volume
 
-A MarkLogic Docker container is always instantiated with a volume (`/var/opt/MarkLogic`).
+A MarkLogic Docker container stores data in `/var/opt/MarkLogic` which should be persistent in Docker managed volume. It is reommended to use named volumes instead of bind mounts as per [Docker documentation](https://docs.docker.com/storage/volumes/).
 
-Run the following command to instantiate a container:
-
-```
-$ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
-     -e MARKLOGIC_INIT=true \
-     -e MARKLOGIC_ADMIN_USERNAME=<insert admin username> \
-     -e MARKLOGIC_ADMIN_PASSWORD=<insert admin password> \
-     PLACEHOLDER-FOR-DOCKER-IMAGE:DOCKER-TAG
-```
-
-Verify volume creation with this command:
+The following command will list previously created volumes:
 
 ```
 $ docker volume ls
 ```
 
-Alternately, you can bind a volume at runtime using the `-v` option:
+The command should output at least two randomly generated volume identifiers from the previous commands.
```

Comment:
I think we should add the following output  as an example snippet afterwards: 

Example output: 
```
DRIVER    VOLUME NAME
local     0f111f7336a5dd1f63fbd7dc07740bba8df684d70fdbcd748899091307c85019
local     1b65575a84be319222a4ff9ba9eecdff06ffb3143edbd03720f4b808be0e6d18
```

Reply:
Furthermore do we want to add an example output after the named volumes have been created?